api/api/utils/license.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from openverse_attribution.license import License
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_license_slug_enum(slug: str) -> tuple[str, License] | None:
```

Comment:
Could we use a namedtuple here to make it clearer how this is being used in the serializers?